packages/x-scheduler-headless/src/utils/recurring-event-utils.ts

```diff
@@ -65,10 +97,9 @@ export function tokenizeByDay(byDay: RecurringEventByDayValue): {
  */
 export function parsesByDayForWeeklyFrequency(
   ruleByDay: RecurringEventRecurrenceRule['byDay'] | undefined,
-  fallback: RecurringEventWeekDayCode[],
```

Comment:
I think it's easier to read that way, and if skips the fallback generation when not needed